doc/langref.html.in

```diff
@@ -9317,6 +9317,31 @@ const c = @cImport({
         <li>Does not support Zig-only pointer attributes such as alignment. Use normal {#link|Pointers#}
         please!</li>
       </ul>
+      <p>When a C pointer is pointing to a struct, deference the C pointer to 
+        access to the struct's fields or member data. That syntax looks like 
+        this: </p>
+        <p>{#syntax#}ptr_to_struct.*.struct_member{#endsyntax#}</p>
+        <p>This is comparable to doing {#syntax#}->{#endsyntax#} in C.</p>
+        <p class="file">c_ptr_test.zig</p>
+        {#code_begin|syntax#}
+const warn = @import("std").debug.warn;
+
+pub fn main() void { // the 'packed' keyword provides C ABI compatability
+    const quick_struct = packed struct {
+        int_member: i32,
+    };
+    var struct_instance = quick_struct {
+        .int_member = 3,
+    };
+    // This is just a syntax example. You should only use C pointers to be
+    // compatible with generated C code.
+    var test_c_ptr: [*c]quick_struct = &struct_instance;
```

Comment:
`[*c]` is for when you don't know if you're pointing to a single item or an array. If you know for sure it's only one, you can use `*` just as well and be C compatible.

Reply:
I removed the code sample. IMO, your wording better expresses the use case for C pointers than the doc's wording.